taming/data/create_data_txt_files.py

```diff
@@ -0,0 +1,24 @@
+import os
+import glob
```

Comment:
This file is not needed. We already have this shell command: /home/elhamod/projects/taming-transformers/analysis/create_file_lists.sh :)